style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
```

Comment:
Id like to see this cover support comments specificly. I think we should use something like https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197 see table at bottom of comment and discussion that follows.